### PR TITLE
Add ability to take license key from package.json

### PR DESCRIPTION
**Pull request by neilstuartcraig**
This is my initial idea to solve #23 
How do you feel about this @runk?

### CODE REVIEW

File: scripts/postinstall.js [hunk 1]
```diff
@@ -4,12 +4,28 @@ const zlib = require("zlib");
...
+}
+
 if (!licenseKey) {
   console.error(`Error: License key is not configured.\n
   You need to signup for a _free_ Maxmind account to get a license key.
   Go to https://www.maxmind.com/en/geolite2/signup, obtain your key and
-  put it in the MAXMIND_LICENSE_KEY environment variable\n`);
+  put it in the MAXMIND_LICENSE_KEY environment variable
+  or in your package.json file (at the root level) like this:
+  "node-geolite2": {
```
Comment (runk) [inline scripts/postinstall.js:26]: Can you please change this to just "geolite2" (since it's the module name on npm)

**Comment by runk** [comment]
Also, it'd be great to add a line to README explaining this alternative option for providing the licence key

**Comment by runk** [comment]
Can you please rebase your branch with master? There were some issues with the env var

**Comment by runk** [comment]
Merging this one into my branch so I can finish this work off

**Comment by neilstuartcraig** [comment]
Sorry @runk - i didn't see your messages yesterday but that's brilliant, thanks so much, this really helps me a out a lot!

**Pull request merged**